back/engines/free/email_campaigns/spec/mailers/previews/email_campaigns/initiative_resubmitted_for_review_mailer_preview.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module EmailCampaigns
+  class InitiativeResubmittedForReviewMailerPreview < ActionMailer::Preview
+    include EmailCampaigns::MailerPreviewRecipient
+
+    def campaign_mail
+      campaign = EmailCampaigns::Campaigns::InitiativeResubmittedForReview.first
+      initiative = Initiative.order(created_at: :asc).first
+      user = User.order(created_at: :asc).first
+      item = Notifications::InitiativeResubmittedForReview.new(post: initiative)
+      activity = Activity.new(item: item)
+      commands = EmailCampaigns::Campaigns::InitiativeResubmittedForReview.new.generate_commands(recipient: user, activity: activity)
```

Comment:
I think we should stop generating a command hash manually and instead, we could call a `generate_commands` method like here.
1. Less boilerplate
2. It tests the mailer in a more integrated way